packages/ui/src/ui/pages/query-tracker/module/query_aco/reducer.ts

```diff
@@ -1,31 +1,37 @@
-import {QUERY_ACO_LIST_LOADING, QUERY_ACO_LIST_SUCCESS, QueryACOActions} from './actions';
+import {QUERY_ACO_LOADING, QUERY_ACO_SUCCESS, QueryACOActions} from './actions';
 
-export interface QueryACOListState {
-    list: string[];
+export interface QueryACOState {
+    cluster_name: string;
+    access_control_objects: string[];
+    supported_features: {access_control: boolean};
     status: 'idle' | 'loading' | 'succeeded' | 'failed';
```

Comment:
Many reducers in the project uses:
```ts
loading: boolean;
loaded: boolean;
```
instead of `status: string` field, lets follow the approach.